tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op_test.cc

```diff
@@ -49,7 +49,10 @@ using ::testing::ElementsAre;
 
 class TRTEngineOpTestBase : public OpsTestBase {
  public:
-  void AddSimpleTrtOp(DataType dtype, int max_cached_engines_count = 1) {
+  void AddSimpleTrtOp(DataType dtype, int max_cached_engines_count = 1,
```

Comment:
Revert all changes in trt_engine_op_test.cc until we have a test for explicit batch mode.